### PR TITLE
readthedocs build fix

### LINKED ISSUE #526: readthedocs build failing due to no index.html generated
On August 1st 2023 readthedocs deprecated builds that do not produce an index.html 

https://blog.readthedocs.com/builds-without-index/

This causes the following error:

Screen Shot 2023-08-03 at 2 31 51 PM

Note [docs.dart.dart.edu ](https://docs.dart.ucar.edu/en/latest/README.html) is still up. This error is for new builds of the documentation - which is why pull requests are failing to build the docs.

Looking into the sphinx versions:
`master_doc` is renamed `root_doc` from sphinx v4+
https://www.sphinx-doc.org/en/master/usage/configuration.html

### PR DESCRIPTION
**Pull request by hkershaw-brown**
## Description:

* Added an explicit target for the welcome page (index.rst). The links to README.rst were relative using the filename, so had to be updated for the rename to index.rst. 
* Using root_doc instead of master_doc to follow latest sphinx conf.py guidance 
* explicit conf.py following readthedocs v2 configuration

### Fixes issue

fixes #526

### Types of changes

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Documentation update

### Documentation changes needed?

- [ ] My change requires a change to the documentation.
  - [ ] I have updated the documentation accordingly.

### Tests
Tested the build on readthedocs:
fix-doc-index branch build: https://readthedocs.org/projects/dart-documentation/builds/21507857/

## Checklist for merging

- [ ] Updated changelog entry
- [ ] Documentation updated
- [ ] Update conf.py

## Checklist for release
- [ ] Merge into main
- [ ] Create release from the main branch with appropriate tag
- [ ] Delete feature-branch

## Testing Datasets

- [ ] Dataset needed for testing available upon request
- [ ] Dataset download instructions included
- [ ] No dataset needed

### CODE REVIEW

File: models/wrf/tutorial/README.rst
Comment (mjs2369) [inline models/wrf/tutorial/README.rst:138]:
```suggestion
   not, you will need to do so now. See :ref:`Getting Started <Welcome page>`
```

File: models/wrf/tutorial/README.rst
Comment (mjs2369) [inline models/wrf/tutorial/README.rst:579]:
```suggestion
DART and was covered in :ref:`Getting Started <Welcome page>`. In
```

File: models/wrf/tutorial/README.rst
Comment (mjs2369) [inline models/wrf/tutorial/README.rst:30]:
```suggestion
understanding the :ref:`Getting Started <Welcome page>` documentation.
```

File: models/clm/tutorial/README.rst
Comment (mjs2369) [inline models/clm/tutorial/README.rst:34]:
```suggestion
understanding the :ref:`Getting Started <Welcome page>` documentation.
```

**Comment by mjs2369** [comment]
@hkershaw-brown everything looks great. I added some suggested changes to make the references to the Welcome page consistent (using "Getting Started" as the name), but feel free to ignore them.

I went through the docs to make sure all the references were working, and they are. So this is ready to merge into main once you either approve/reject the suggested changes, and then I'll approve the PR immediately afterwards

**Comment by hkershaw-brown** [comment]
@mjs2369  I can't see your suggestions.  

Do you want to change the name of the label:
`.. _Welcome page:` to `.._Getting Started`

**Review by mjs2369 — COMMENTED** [review]
@hkershaw-brown this should reveal my suggestions, sorry about that

File: models/wrf/tutorial/README.rst
Comment (hkershaw-brown) [reply models/wrf/tutorial/README.rst:30]: I think this is to be clear that a user should read the DART getting started documentation (rather than just WRF).

File: models/clm/tutorial/README.rst
Comment (hkershaw-brown) [reply models/clm/tutorial/README.rst:34]: Same comment: I think this is to be clear that a user should read the DART getting started documentation (rather than just WRF).

File: models/wrf/tutorial/README.rst
Comment (mjs2369) [reply models/wrf/tutorial/README.rst:30]: Yeah that's a good point

**Review by mjs2369 — APPROVED** [review]

**Pull request merged**
